sql/patch.sql

```diff
@@ -47,3 +47,7 @@
 
 --  #EndIf
 --    all blocks are terminated with and #EndIf statement.
+
+#IfMissingColumn openemr_postcalendar_events uuid
+ALTER TABLE `openemr_postcalendar_events` ADD `uuid` binary(16) DEFAULT NULL;
+#EndIf
```

Comment:
I think we only use this during the patch process and not for codebase commits but could be mistaken.

Reply:
K, removed the changes.